menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala

```diff
@@ -53,6 +55,51 @@ class RunMongoRepository @Autowired()(mongoDb: MongoDatabase)
     excludeId()
   ))
 
+  private def getTodaysFilter() = {
+    val date = LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))
+    regex("startDateTime", s"^$date")
+  }
+
+  private def getTodayRunsCount(filters: List[Bson]): Future[Int] = {
+    val pipeline = Seq(
+      filter(and((getTodaysFilter :: filters) :_*)),
+      Aggregates.count("count"))
+    collection.aggregate[BsonDocument](pipeline).headOption().map({
+        case Some(doc) => doc.getInt32("count").getValue
+        case None => 0
+    })
+  }
+
+  def getTodaysRuns(): Future[Int] = {
+    getTodayRunsCount(List())
+  }
+
+  def getTodaysSuccessfulRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "allSucceeded")))
+  }
+
+  def getTodaysFailedRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "failed")))
+  }
+
+  def getTodaysStdSuccessRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "stageSucceeded")))
+  }
+
+  def getTodaysRunningRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "running")))
+  }
+
+  def getTodaysSuccessWithErrors(): Future[Int] = {
```

Comment:
Having this another status is a great idea, as it will draw more attention to the errors. Maybe it would be better to implement it in std/cnfrm jobs, so that Menas UI and the database have the same set of statuses? Otherwise, it may be misleading when tracing the errors. Alternatively, "Completed jobs with data errors", could be displayed as a separate metrics, not to be confused with job execution status... What are your thoughts on this?

Reply:
@aaishaosman @yruslan 

So I guess we'd like to have as much metadata as possible. I believe in future we may need to show stuff like break down of errors etc.

Reply:
Yeah, breakdown of errors is something I'd expect in the future. I'm wondering what monitoring guys are going to say they want.